content/programming-cookbook/vim-and-neovim/basic-search.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Basic Search in VIM/NVIM
+showMetadata: true
+editable: true
+---
+
+> How to search in vim?
+
+it's very easy,you need to be on `Normal` mode 
```

Comment:
Please make all complete English sentences by starting with a single capital and end with period.